src/test/java/com/opentable/extension/BodyTransformerTest.java

```diff
@@ -18,10 +18,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
We prefer to specify each import instead of using .*. It helps reduce import bloat, helps with debugging class traces, and helps prevent unforeseen collisions.